pages/education/school-resources.md

```diff
@@ -111,19 +111,23 @@ relatedlinks:
       description:
 ---
 
-Resources for schools is a one-stop shop for School Certifying Officials (SCOs) and school administrators assisting students pursuing education and training with VA benefits. Find trainings, resources, guides, and information on GI Bill programs to support military-connected students.
+<div>
+  <p class="va-introtext">
+    Resources for schools is a one-stop shop for School Certifying Officials (SCOs) and school administrators assisting students pursuing education and training with VA benefits. Find trainings, resources, guides, and information on GI Bill programs to support military-connected students.
+  <p>
+</div>
 
 ### On this page
 
-[Training and guides](#training-and-guides)
+- [Training and guides](#training-and-guides)
 
-[Upcoming events](#upcoming-events)
+- [Upcoming events](#upcoming-events)
 
-[Policies and procedures](#policies-and-procedures)
+- [Policies and procedures](#policies-and-procedures)
 
-[Resources to support students](#resources-to-support-students)
+- [Resources to support students](#resources-to-support-students)
 
-### Key Resources for SCOs
+### Key Resources for SCO
```

Comment:
This needs to be put back to SCOs, otherwise looks good.

Reply:
Added the s